cfy_manager/components/validations.py

```diff
@@ -90,6 +91,45 @@ def _validate_supported_distros():
         )
 
 
+def _validate_private_ip():
+    logger.info('Validating private IP address...')
+    private_ip = config[MANAGER][PRIVATE_IP]
+    all_addresses = set()
+    found = False
+    all_interfaces = netifaces.interfaces()
+    for interface in all_interfaces:
```

Comment:
Might as well not bother with the extra variable and just do
`for interface in netifaces.interfaces()`